core/src/main/scala/org/apache/spark/util/ThreadUtils.scala

```diff
@@ -274,12 +274,19 @@ private[spark] object ThreadUtils {
    * @return new collection in which each element was given from the input collection `in` by
    *         applying the lambda function `f`.
    */
-  def parmap[I, O](in: Seq[I], prefix: String, maxThreads: Int)(f: I => O): Seq[O] = {
+  def parmap[I, O, Col[X] <: TraversableOnce[X]]
```

Comment:
Oh, I should be clear: `TraversableOnce` also doesn't exist in 2.13.
I don't see the use case for returning the same collection, not in this limited utility method?

Reply:
Hm, `CanBuildFrom` doesn't exist in 2.13 either. Then we have to write two versions for 2.12 and 2.13. Hence, I'm okey to just use `Seq` instead. Adding `asInstanceOf` if necessary is better than two versions of codes.

Reply:
Thank you for the investigation and the conclusion, @zsxwing and @srowen .